third_party/pcsc-lite/naclport/server/src/reader-tracker.js

```diff
@@ -150,81 +281,391 @@ ReaderTracker.prototype.readerFinishAddListener_ = function(message) {
 
   var returnCodeHex = GSC.DebugDump.dump(returnCode);
 
-  var value = new ReaderInfo(name, ReaderStatus.SUCCESS);
-
-  var text =
-      'Finished initializing reader ' + name + ' (port ' + port +
-      ', device ' + device + ') with return code ' + returnCodeHex;
+  var logMessage =
+      'The reader "' + name + '" (port ' + port + ', device "' + device +
+      '") was initialized with return code ' + returnCodeHex;
+  if (returnCode === 0)
+    this.logger_.info(logMessage);
+  else
+    this.logger_.warning(logMessage);
```

Comment:
Use a consistent bracing style (a few lines below you use braces, and here you don't).